src/ms/validation.php

```diff
@@ -112,23 +112,5 @@
     |
      */
 
-    'attributes'           => [
```

Comment:
actually, it was from my previous pull request. it was from my previous project. i forgot to remove it. sorry for that.